internal/multigitter/cmd.go

```diff
@@ -0,0 +1,23 @@
+package multigitter
+
+import (
+	"context"
+	"fmt"
+	"github.com/lindell/multi-gitter/internal/scm"
+	"os"
+	"os/exec"
+)
+
+func prepareScriptCommand(ctx context.Context, repo scm.Repository, workDir string, scriptPath string, arguments []string, dryRun bool) (cmd *exec.Cmd) {
```

Comment:
You are right in that run/print should be refactored to use more of the same code, and there are actually more that could be shared!

But it should not be a focus of this PR to keep the focus of each PR to a minimum.

Reply:
So you want me to take this out and split into another PR?

Reply:
Yes, if you want to do additional improvement on that part 🙂 
Otherwise, remove those changes and create a issue about refactoring it.